save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ProjectController.kt

```diff
@@ -247,8 +247,8 @@ class ProjectController(
             }
         }
         .map { updatedProject ->
-            val (_, projectStatus) = projectService.getOrSaveProject(updatedProject)
-            ResponseEntity.ok(projectStatus.message)
+            projectService.updateProject(updatedProject)
```

Comment:
Do we have separate logic for saving new projects? I remember something about this logic being shared for two operations